<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryTreeUI</class>
 <widget class="QWidget" name="BinaryTreeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>逆波兰树</string>
   </property>
   <widget class="QPushButton" name="Button_InsertBiTreeNode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>67</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>创建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_CleanBiTreeNode">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>67</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BinaryTreeDataEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入中缀表达式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>逆波兰式值</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_CleanBiTreeNode</sender>
   <signal>clicked()</signal>
   <receiver>BinaryTreeUI</receiver>
   <slot>on_binarytree_delete_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_InsertBiTreeNode</sender>
   <signal>clicked()</signal>
   <receiver>BinaryTreeUI</receiver>
   <slot>on_binarytree_create_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_binarytree_create_btn_clicked()</slot>
  <slot>on_binarytree_delete_btn_clicked()</slot>
  <slot>on_binarytree_node_delete_btn_clicked()</slot>
  <slot>on_binarytree_SimulationTree_creat_btn_clicked()</slot>
  <slot>on_bigroottree_create_btn_clicked()</slot>
  <slot>on_bigroottree_insert_btn_clicked()</slot>
  <slot>on_bigroottree_remove_btn_clicked()</slot>
  <slot>on_Btree_create_btn_clicked()</slot>
  <slot>on_Btree_insert_btn_clicked()</slot>
  <slot>on_Btree_remove_btn_clicked()</slot>
  <slot>on_Btree_destroy_btn_clicked()</slot>
  <slot>on_binarytree_node_search_btn_clicked()</slot>
  <slot>on_Btree_search_btn_clicked()</slot>
  <slot>on_binarytree_exchangecreate_btn_clicked()</slot>
  <slot>on_binarytree_node_insert_btn_clicked()</slot>
 </slots>
</ui>
